### PR TITLE
adding a link -  guidelines from the official Android docs

### PR DESCRIPTION
**Pull request by tsonevn**

### CODE REVIEW

File: docs/tooling/android-virtual-devices.md [hunk 1]
```diff
@@ -7,91 +7,8 @@ slug: android-emulators
...
-
-To launch a NativeScript application on all connected devices simultaneously, simply ignore the `--device <id>` flag.
-
-```
-tns run android
-```
-
-
+Using Android emulator a.k.a. Android Virtual Device (AVD) eases the development process by providing an option to test mobile applications on various virtual devices. By default, the Android SDK does not provide an instance of an AVD, so it is required to make some additional steps to allow its use on your machine. 
+Android provides an article in their official documentation called [Create and Manage Virtual Devices](https://developer.android.com/studio/run/managing-avds.html), where it is covered the processes of downloading, creating and using one Android Emulator via Android Studio.
```
Comment (petekanev) [inline docs/tooling/android-virtual-devices.md]: Could we reword this to: "Follow the official documentation on [Creating and Managing Virtual Devices](https://developer.android.com/studio/run/managing-avds.html), where the process of downloading, setting up, and using Android Emulators via Android studio is covered."

Comment (tsonevn) [reply docs/tooling/android-virtual-devices.md]: yes, it sounds much better. I will change it

**Review by etabakov — APPROVED** [review]
